types/serverless/serverless-tests.ts

```diff
@@ -327,8 +327,7 @@ const awsServerless: Aws.Serverless = {
         }
     },
     package: {
-        include: ['testinclude'],
-        exclude: ['testexclude'],
+        patterns: ['!testpatternexclude', 'testpatterninclude'],
```

Comment:
Let's keep `include` & `exclude` in the test, as those properties are still valid and so should be allowed in the types :)

Reply:
Done :+1: